Framework/DataHandling/src/LoadILLReflectometry.cpp

```diff
@@ -38,19 +38,19 @@ struct PeakInfo {
  *  @param x an angle in degrees
  *  @return the angle in radians
  */
-constexpr double inRad(const double x) { return x * M_PI / 180; }
+constexpr double inRad(const double x) { return x * M_PI / 180.; }
```

Comment:
I would suggest changing these names to degToRad or similar, else it does not really explain what they are doing